Sources/StreamChatUI/ChatChannelList/ChatChannelListVC+SwiftUI_Tests.swift

```diff
@@ -0,0 +1,66 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+import StreamChatTestTools
+@testable import StreamChatUI
+import XCTest
+
+@available(iOS 13.0, *)
+class ChatChannelListVCComponent_Tests: XCTestCase {
```

Comment:
This is not enough, unfortunately. The test case should inherit from `iOS13TestCase`.

Reply:
Good point, adjusted.